framework/wazuh/core/indexer/models/events.py

```diff
@@ -207,6 +207,6 @@ def get_index_name(self) -> str:
         return VULNERABILITY_INDEX
 
 
-class Events(BaseModel):
-    """Object holding a list of events of any type."""
+class StatefulEvents(BaseModel):
+    """Object holding a list of stateful events of any type."""
```

Comment:
I have missed requesting in the review to apply what we discussed in the meeting. This class should be part of the `apis/comms_api/models/events.py` file and could be redefined to have a `StatefulEvent` list similar to how `StatelessEvents` was defined.

Reply:
We concluded that Fede would make this change, but I will do it in this one anyway since it's tiny